src/OpenColorIO/CMakeLists.txt

```diff
@@ -100,6 +100,10 @@ if(MSVC)
 	set_target_properties(OpenColorIO PROPERTIES COMPILE_FLAGS "/wd4275")
 endif()
 
+set_target_properties(OpenColorIO PROPERTIES
```

Comment:
Fixed the *nix platforms library name. Unfortunately, it's still not working for Windows.

Reply:
We should only need to set one of these - https://cmake.org/cmake/help/v3.10/prop_tgt/SOVERSION.html

The reason it is failing on windows is due to the addition of the dev flag into the SOVERSION string. On windows this must be "MAJOR.MINOR"

Reply:
I tried and it's still not working. I tried ***2.0.0dev***, ***2.0.0*** and ***2.0***.